src/_gettsim/parameters/ges_rente.yaml

```diff
@@ -975,6 +976,109 @@ abolishment_cohort_rente_für_arbeitsl:
   note: https://www.sozialgesetzbuch-sgb.de/sgbvi/237.html
   1980-01-01:
     scalar: 1952
+altergrenze_arbeitsl_abschlagsfrei_vertrauensschutz:
+  name:
+    de: >-
+      Altersgrenze für Altersrente wegen Arbeitslosigkeit mit  Vertrauensschutz
+      (abschlagsfrei).
+    en: >-
+      Age threshold for pension for unemployed (without deductions)
+      with Vertrauensschutz
+  description:
+    de: >-
+      Frühesmögliches Alter ab dem die Altersrente für Arbeitslose, für die der
+      Vertrauensschutz gilt abschlagsfrei bezogen werden kann.
+    en: >-
+      Earliest possible age to receive pension for unemployed without  deductions
+      under legitimate expectations.
+  reference: § 237 SGB VI
+  note: >-
+    https://www.gesetze-im-internet.de/sgb_6/__237.html
+  type: piecewise_linear
+  1980-01-01:
```

Comment:
Two comments:
1. Can we use the correct starting date, i.e., when the relevant section of § 237 SGB VI was added?
2. IIUC, this basically implements this table?

   ![image](https://github.com/iza-institute-of-labor-economics/gettsim/assets/158456/5bf96f17-51bc-49a9-ab73-76c41ba2cffd)

   If so, there should be a simpler way of implementing this than piecewise linear, I think. Happy to discuss.

Reply:
changed to 2005.
Yes, lets discuss